doc/api/vm.md

```diff
@@ -295,6 +295,55 @@ console.log(globalVar);
 // 1000
 ```
 
+## `vm.measureMemory([options, [contextifiedObject]])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Measure the memory used by the current execution context or a specified context.
```

Comment:
It should likely be stressed that this only returns the memory usage that is specifically known to V8.

Reply:
I've added emphasis about this in several places in the doc